.github/ISSUE_TEMPLATE/BUG_BACKEND.md

```diff
@@ -0,0 +1,29 @@
+---
+name: ⚠️ Report Backend Bug
+about: Report a Backend bug on Kubeflow Pipelines
+title: "[backend] <Bug Name>"
+labels: kind/bug, area/backend
+---
+
+### Environment
+
+*  How did you deploy Kubeflow Pipelines (KFP)?
+<!-- See [an introduction of all options](https://www.kubeflow.org/docs/pipelines/installation/overview/). -->
+*  KFP version: <!-- See version number shows on bottom of KFP UI left sidenav. -->
+*  KFP SDK version: <!-- Please attach the output of this shell command: $pip list | grep kfp -->
+
+
+### Steps to reproduce
+
+<!--
+How can we make the problem occur?
+This could be a description, code snippet, log output, screenshot, etc.
+-->
+
+### Expected result
+
+<!-- What should the correct behavior be? -->
+
+### Materials and Reference
+
+<!-- Help oncaller to debug by providing sample code, background context, links to reference, etc. -->
```

Comment:
@zijianjoy I learned from argo that, we can add a section at the end of each bug to say sth like:

```
please thumbs up on the issue if you care about it, we prioritize by thumbs up counts
```

That makes it more obvious a way for users to tell importance of a bug to us.

ref: https://github.com/argoproj/argo-workflows/issues

Reply:
That is a very good footprint to add! Thank you for suggesting it, I applied it to both bugs and feature requests templates.